File: app/src/main/java/com/frama/miserend/hu/base/BaseFragment.java
package com.frama.miserend.hu.base;

import android.content.Context;
import android.support.v4.app.Fragment;

import dagger.android.support.AndroidSupportInjection;

/**
 * Created by Balazs on 2018. 02. 19..
 */

public class BaseFragment extends Fragment {
    @Override
    public void onAttach(Context context) {
        AndroidSupportInjection.inject(this);
        super.onAttach(context);
    }
}

File: app/src/main/java/com/frama/miserend/hu/base/BaseFragment.kt
package com.frama.miserend.hu.base

import android.content.Context
import android.support.v4.app.Fragment

import dagger.android.support.AndroidSupportInjection

/**
 * Created by Balazs on 2018. 02. 19..
 */

open class BaseFragment : Fragment() {

    override fun onAttach(context: Context?) {
        AndroidSupportInjection.inject(this)
        super.onAttach(context)
    }
}
